### PR TITLE
[Darkside] Select CSS update

### PR DESCRIPTION
**Pull request by KenAJoh**

### CODE REVIEW

File: @navikt/core/css/darkside/form/select.darkside.css
Comment (HalvorHaugan) [inline @navikt/core/css/darkside/form/select.darkside.css:1]: Is readonly supposed to have hover effect in forced-colors mode?
image

File: @navikt/core/css/darkside/form/select.darkside.css [hunk 1]
```diff
@@ -1,115 +1,113 @@
 .navds-select__input {
   appearance: none;
-  background-color: var(--ac-select-bg, var(--a-surface-default));
-  border-radius: var(--a-border-radius-medium);
-  border: 1px solid var(--ac-select-border, var(--a-border-default));
-  color: var(--ac-select-text, var(--a-text-default));
+  background-color: var(--ax-bg-input);
+  border-radius: var(--ax-border-radius-medium);
```
Comment (HalvorHaugan) [inline @navikt/core/css/darkside/form/select.darkside.css:4]: Is this the correct border-radius?

Comment (KenAJoh) [reply @navikt/core/css/darkside/form/select.darkside.css:4]: Yes, the larger radius is currently just experimental in Figma
Screenshot 2024-11-25 at 16 45 36

File: @navikt/core/css/darkside/form/select.darkside.css
Comment (KenAJoh) [reply @navikt/core/css/darkside/form/select.darkside.css:1]: Fixed 🔨

**Review by HalvorHaugan — APPROVED** [review]

**Pull request merged**